Assets/Scripts/Board.cs

```diff
@@ -1,103 +1,90 @@
-using System.Collections;
-using UnityEngine;
-
+using System.Collections; // IEnumerator
+using System.Collections.Generic; // IEnumerable
 using System.Linq;
 
+using UnityEngine;
+
 public class Board : MonoBehaviour
 {
 	private GameManager GM;
 	public ColorManager ColorManager;
 
 	public Material disolve;
 
-	public bool foundRow = false;
-	public float disolveTimer = 0;
-
 	public GameObject PlayerBase;
 
+	[SerializeField]
 	private int rotationState = 0;
 	private GameObject[,] slots;
 
+	// Move to settings
+	private const int ANIMATION_DURATION = 1;
+	private const int N_ROWS = 13;
+
 	void Awake()
 	{
 		GM = GameManager.Instance;
 
-		slots = new GameObject[12, Geometry.PERIODICITY];
+		slots = new GameObject[N_ROWS, Geometry.PERIODICITY];
 	}
 
-	public bool IsEmpty(Slot slot)
+	public void CheckForCompleteRows()
```

Comment:
Check this function :)

Reply:
noice. du verfolgst schon immer noch ein besseren Ansatz. Find ich gut. guter input